config/overlays/development/kustomization.yaml

```diff
@@ -2,4 +2,5 @@ bases:
   - ../../default
 
 patches:
+  - configmap/inferenceservice.yaml
```

Comment:
can you name this file "patch" of some sort?

Reply:
Done.

Reply:
@ellis-bigelow Anymore comments on this?